docker-bake.hcl

```diff
@@ -1,10 +1,6 @@
 # VARIABLES
 variable "REGISTRY" {
-  default = "ghcr.io"
-}
-
-variable "REPOSITORY" {
-  default = "layr-labs/eigenda"
+  default = "ghcr.io/layr-labs/eigenda"
```

Comment:
Do you want to add the ecr repo as a comment for reference?

Reply:
There is ecr for each account dev/preprod/testnet/mainnet, but mainly I don't want to leak internal account ecr urls in oss repo

Reply:
fair point